stream/server/src/main/java/org/apache/bookkeeper/stream/server/StorageServer.java

```diff
@@ -256,10 +260,19 @@ public static LifecycleComponent buildStorageServer(CompositeConfiguration conf,
 
             // Build http service
             if (bkServerConf.isHttpServerEnabled()) {
+                MetadataBookieDriver metadataDriver = BookieResources.createMetadataDriver(bkServerConf,
```

Comment:
are we going to create an additional connection to ZooKeeper ?

Reply:
AFAICT, there is an internal cache in MetadataDrivers so if there is one already it will be reused